docs/howto/openstack/keystone/app-creds.md

```diff
@@ -0,0 +1,277 @@
+```
+
+You may select specific columns...
+
+```console
+$ openstack application credential list -c ID -c Name -c Expires\ At
+
++----------------------------------+----------------+----------------------------+
+| ID                               | Name           | Expires At                 |
++----------------------------------+----------------+----------------------------+
+| 98a2804b2d294b6784ae30ec67697382 | angry_lamarr   | None                       |
+| a39676b7b0bd4c0c84fa06bdde44b090 | elated_jones   | None                       |
+| 43649f148bbe46ab8f1d745eea095db9 | goofy_dijkstra | 2024-08-31T23:59:59.000000 |
++----------------------------------+----------------+----------------------------+
+```
+
+...or choose a different output format, like JSON:
+
+```console
+$ openstack application credential list -f json
+
+[
+  {
+    "ID": "98a2804b2d294b6784ae30ec67697382",
+    "Name": "angry_lamarr",
+    "Project ID": "dfc700467396428bacba4376e72cc3e9",
+    "Description": null,
+    "Expires At": null
+  },
+  {
+    "ID": "a39676b7b0bd4c0c84fa06bdde44b090",
+    "Name": "elated_jones",
+    "Project ID": "dfc700467396428bacba4376e72cc3e9",
+    "Description": null,
+    "Expires At": null
+  },
+  {
+    "ID": "43649f148bbe46ab8f1d745eea095db9",
+    "Name": "goofy_dijkstra",
+    "Project ID": "dfc700467396428bacba4376e72cc3e9",
+    "Description": "Direct connection to Iqaluit",
+    "Expires At": "2024-08-31T23:59:59.000000"
+  }
+]
+```
+
+You can view specific application credentials like so:
+
+```console
+$ openstack application credential show goofy_dijkstra
+
++--------------+----------------------------------------------------------+
+| Field        | Value                                                    |
++--------------+----------------------------------------------------------+
+| description  | Direct connection to Iqaluit                             |
+| expires_at   | 2024-08-31T23:59:59.000000                               |
+| id           | 43649f148bbe46ab8f1d745eea095db9                         |
+| name         | goofy_dijkstra                                           |
+| project_id   | dfc700467396428bacba4376e72cc3e9                         |
+| roles        | swiftoperator reader member creator load-balancer_member |
+| system       | None                                                     |
+| unrestricted | False                                                    |
+| user_id      | e50719d594f84f16ad13f88da540f762                         |
++--------------+----------------------------------------------------------+
+```
+
+No matter how you choose to list or view application credentials, secrets **are never** displayed.
+
+## Restricted vs unrestricted credentials
+
+By default, application credentials are created as being *restricted*; that is why in the `openstack application credential show` outout above, the `unrestricted` parameter is set to `False`.
```

Comment:
Typo. And also, I think that instead of the semicolon, you can just put two sentences. And I'd inject one more comma:
```suggestion
By default, application credentials are created as being *restricted*.
That is why, in the `openstack application credential show` output above, the `unrestricted` parameter is set to `False`.
```